src/com/yetanalytics/lrs_admin_ui/views/browser.cljs

```diff
@@ -151,6 +151,15 @@
           [:img {:src "images/icons/next.svg"
                  :width "30px"}]])]]]))
 
+(defn refresh-button []
+  (when @(subscribe [:browser/get-credential])
+    [:a.icon-refresh
+     {:href "!#"
+      :on-click (fn [e]
+                  (fns/ps-event e)
+                  (dispatch [:browser/refresh]))}
+     "Refresh"]))
```

Comment:
Prob want to use use localization / override system since this is visible text.

Reply:
oh right

Reply:
done https://github.com/yetanalytics/lrs-admin-ui/pull/79/commits/a294481677394430a2bcc2a094d341b723456a59